spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cassandra/CassandraProperties.java

```diff
@@ -26,12 +26,14 @@
 import com.datastax.driver.core.policies.RetryPolicy;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.data.cassandra.config.SchemaAction;
```

Comment:
Missed this one to revert it to `String` after using the relaxed property resolver. Changed it now.